### ISSUE TITLE
Issue with k8s.io/docs/tasks/run-application/run-stateless-application-deployment

### ISSUE THREAD

**Issue by cvandesande** [kubernetes/website#19545]
This is a Bug Report

Problem:
This is similar to the the issue here: 
https://github.com/kubernetes/website/issues/14514

Though I believe I know the cause...the nginx:1.7.9 container will crash on modern kernels with vsyscall disabled.  For example on Debian buster kernel 5.4.0-0.bpo.2-amd64 this happens:
```
[ 8470.383161] nginx[25332] vsyscall attempted with vsyscall=none ip:ffffffffff600400 cs:33 sp:7fff541ba6a8 ax:ffffffffff600400 si:49a159 di:0
[ 8470.383165] nginx[25332]: segfault at ffffffffff600400 ip ffffffffff600400 sp 00007fff541ba6a8 error 15
[ 8470.383166] Code: Bad RIP value.
```

This issue is extremely annoying to troubleshoot. There are no container logs indicative of any error and kubectl describe pod/nginx-xxxx shows no errors other than these events:
```
9m17s       Normal    Created                   pod/nginx-deployment-54f57cf6bf-xqpdp    Created container nginx
9m16s       Normal    Started                   pod/nginx-deployment-54f57cf6bf-xqpdp    Started container nginx
9m17s       Normal    Pulled                    pod/nginx-deployment-54f57cf6bf-xqpdp    Container image "nginx:1.7.9" already present on machine
47s         Warning   BackOff                   pod/nginx-deployment-54f57cf6bf-xqpdp    Back-off restarting failed container
```

This is the previous bug's description which is true, as is the recommended solution.

The tutorial mentions to create a deployment with nginx version 1.7.9 - doing so will result in a CrashLoopBackoff of the pods.

Proposed Solution:
Using a different version, e.g. 1.15 will result in a running pod.

Page to Update:
https://kubernetes.io/docs/tasks/run-application/run-stateless-application-deployment/

**Comment by sftim** [comment]
This seems worth fixing at some point.
/priority backlog

**Comment by tengqm** [comment]
#19198 has resolved this issue, though #19622 wasn't merged.
Please feel free to open if this is still an issue.
/close
